tests/requirements-py3.txt

```
@@ -3,7 +3,6 @@ attrs
 dataclasses; python_version == '3.6'
 mitmproxy; python_version >= '3.7'
 mitmproxy >= 4, < 5; python_version >= '3.6' and python_version < '3.7'
```

Just in case, a suggestion:

```suggestion
mitmproxy >= 4.0.4, < 5; python_version >= '3.6' and python_version < '3.7'
```

There is a CVE-2018-14505 on earlier mitmproxy versions.

---

It got reverted back after the force-push :)

---

:facepalm: